source/utilities.tex

```diff
@@ -35,13 +35,20 @@
 \rSec1[utility]{Utility components}
 
 \pnum
+\begin{note}
 This subclause contains some basic function and class templates that are used
 throughout the rest of the library.
+\end{note}
 
 \indexhdr{utility}%
 \indexlibrary{\idxcode{rel_ops}}%
 \rSec2[utility.syn]{Header \tcode{<utility>} synopsis}
 
+\pnum
+The header \tcode{<utility>} defines several types and function templates
+that are described in this Clause. It also defines the template \tcode{pair}
+and various function templates that operate on \tcode{pair} objects.
```

Comment:
What on earth is the special call-out to `pair` achieving here? The header also defines `in_place_index` and `index_sequence_for`. Maybe this sentence came from an older time when `pair` really was the only class template here?

Reply:
This is pre-existing text that just got moved (from below the synopsis). There are similar introductory notes for most of the header synopses. If you feel this particular intro text warrants improvement, feel free to suggest something in a separate pull request.

Reply:
OK, let me just keep this open so as to not forget.

@zygoloid, @jwakely: do you think we should list all the other class templates explicitly in this strange little sentence, or do something more radical (such as removing it altogether)?

Reply:
I'd prefer we just drop this entirely. If someone wants to know what's in <utility>, that's what the synopsis is for. I don't see any point in including a paragraph that says "The header <utility> includes the stuff we say it includes in the synopsis."

Reply:
Maybe we could ditch this and move the hanging paragraph into this section, replacing "This subclause" with "The header \<utility>".